Integrations/integration-ArcSightLogger.yml

```
@@ -540,15 +540,22 @@ script:
             var keys = [];
             var fields = xmlObject.fields;
             var entries = xmlObject.results;
+            var isDateField = [];
             fields.forEach(function(field){
                 if(field.name){
                     keys.push(field.name.replace(/\s/g, '').replace(/^\_/,''));
+                    isDateField.push(field.type == 'date');
                 }
             });
             entries.forEach(function(entry){
                 if (entry.length == keys.length){
                     var newEntry = {};
                     for (var i = 0; i<  entry.length; i++){
+                        if (isDateField[i]){
```

Let's make sure we have an int
```suggestion
                        if (isDateField[i] && !isNaN(entry[i])) {
```